examples/08-averaging/01-average_across_bodies.py

```diff
@@ -89,7 +97,10 @@ def average_across_bodies(analysis):
     min_max.inputs.fields_container.connect(stresses)
     max_val = min_max.outputs.field_max()
 
-    mesh.plot(stresses)
+    # Uncomment this block to obtain the plot
```

Comment:
@vsousacosta Does this mean the plotting function does not work correctly?

Reply:
Actually, the plotting function works well. The problem apparently has something to do with the document builder, as the plotting issue only happens when building the docs (it works normally when running the code). So, in order to avoid it, I commented the mesh.plot() functions and inserted screenshots of the images I needed.

Reply:
Still, this means there is a bug when generating the plots for the doc, which should be the way to do it. This might have something to do with the package versions used (Pyvista, vtk, etc...) when building the doc vs when running it locally, or with the backend used.